modules/jtframe/devops/docker-installer.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+# This script will remove your existing Docker installation and
+# will do a fresh Docker installation, including official plugins
+# such as composer and buildx (required for building Jotego images).
+# It will also set Docker in rootless mode so you won't need to use
+# sudo, either interactively or within scripts.
+# The script is architecture-independent.
+
+set -e
+
+echo "🚀 Docker installation on Ubuntu for Jotego devops..."
+
+DOCKER_GPG_DIR="/etc/apt/keyrings"
+DOCKER_GPG="$DOCKER_GPG_DIR/docker.gpg"
+ARCH=$(dpkg --print-architecture)
+RELEASE=$(lsb_release -cs)
+
+# 1. Remove any existing Docker installation
```

Comment:
make each one of these sections a function with a descriptive name and remove the comments at the beginning of each one